health/api/v1/regions.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import json
+
+import flask
+import requests
+
+
+regions = flask.Blueprint("regions", __name__)
+
+
+def get_blueprints():
+    return [
+        ["/regions", regions],
+    ]
+
+
+@regions.route("/")
+def get_regions():
+
+    query = {
+        "size": 0,
+        "aggs": {
+            "regions": {
+                "terms": {"field": "region"}
```

Comment:
this will return only up to 10 regions, based on docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#_size